GarfieldBot/Bot.py

```diff
@@ -168,6 +168,23 @@ def send_message(self, channel: Union[Channel, str], text: str) -> None:
             channel=channel,
             text=text
         )
+    
+    def send_to_thread(self, channel: str, thread_ts: str, text: str) -> None:
```

Comment:
```suggestion
    def send_to_thread(self, channel: Union[Channel, str], thread_ts: str, text: str) -> None:
```

Reply:
Parameter should specify that it can take a channel or a str

Reply:
Resolved in https://github.com/lavenduale/GarfieldBot/commit/e675ab20cfefc17da0ec553e3e6705596dbaa26f